lib/errors/validation-error.ts

```diff
@@ -195,11 +205,13 @@ export class ValidationErrorItem {
 }
 
 /**
- * Validation Error. Thrown when the sequelize validation has failed. The error contains an `errors` property,
- * which is an array with 1 or more ValidationErrorItems, one for each validation that failed.
+ * Thrown when the sequelize validation has failed. The error contains an `errors` property,
+ * which is an array with 1 or more [ValidationErrorItem](/esdoc/class/lib/errors/validation-error.ts~ValidationErrorItem.html), one for each validation that failed.
  *
- * @param message Error message
- * @param errors Array of ValidationErrorItem objects describing the validation errors
+ * | parameter | description |
```

Comment:
Same here, this should be jsdoc instead of markdown

Reply:
I've tried jsdoc but looks like it's accept markdown instead of jsdoc, please kindly check :) @ephys

Reply:
The jsdoc has no effect on npm run docs, so i think it's should be markdown.